src/providers/googleProvider.ts

```diff
@@ -18,11 +42,14 @@ export interface RawResult {
   }[];
   formatted_address: string;
   geometry: {
-    location: LatLng;
+    location: {
+      lng: Function;
+      lat: Function;
+    };
     location_type: string;
     viewport: {
-      northeast: LatLng;
-      southwest: LatLng;
+      Ab: LatLng;
+      Ra: LatLng;
```

Comment:
This is something else that I don't quite understand - when I look at the response in the "Network" tab, it looks essentially identical to the response returned via the existing method, which you can see in src/providers/__tests__/googleResponse.json (one of the `"types"` is missing, but that's pretty inconsequential) - so the keys under `viewport` are `northeast` and `southwest`, each with `lat` and `lng` keys under them. However, when the result is accessed here in the code, the format is somewhat different - instead of `northeast` and `southwest`, we have `Ab` and `Ra` - where `Ab` appears to hold latitudes and `Ra` appears to hold longitudes. I'm not sure why that arrangement would change, and why the data here would appear different than the data in the response in the "Network" tab.

Reply:
Must be something in the way the imported Google Maps JS API handles preparing the data for the callback function...